src/test/e2e/helpers/database-init.ts

```diff
@@ -86,6 +87,12 @@ export default async function init(
     getLogger: LogProvider = noLoggerProvider,
     configOverride: Partial<IUnleashOptions> = {},
 ): Promise<ITestDb> {
+    if (isNotSnakeCase(databaseSchema)) {
+        throw new Error(
+            `db init database schema must be snake case, was: ${databaseSchema}`,
+        );
+    }
```

Comment:
I added this because dbInit schemas must be snake case. If you try to call it with hyphons it will fail silently: `dbInit('my-schema')`

Now you will get a clear error message if you make this mistake, and don't spend 30 minutes debugging silly things.